### DISCUSSION TITLE
Adding CDs to the main rotation

### DISCUSSION THREAD

**Discussion by Blazindragon1103** [Hekili/hekili#1723] [Q&A]
Hello, I would like to add items/abilities to the main rotation.  There use to be a list type feature where you could hit red X's to add and take away, but I can seem to find it.  Thank you for the help.

**Comment by maxiixam** [comment]
It's in the Spec Toggles menu.

![image](https://user-images.githubusercontent.com/4550881/199065537-8477e9d6-0f98-4c45-ae40-f45bb1b80ffc.png)

**Reply by Blazindragon1103** [reply]
Thank you.  I'm going to go be in my box of shame in the corner for not remembering this.
